resources/zh/cemu.po

```diff
@@ -1493,337 +1849,321 @@ msgstr "更新提醒"
 
 #: MainWindow.cpp
 msgid "&Graphic packs"
-msgstr "图形包"
-
-#: MainWindow.cpp
-msgid "&GPU buffer cache accuracy"
-msgstr "GPU缓冲区缓存精度"
+msgstr "&图形插件"
 
 #: MainWindow.cpp
 msgid "&Separate GamePad view"
-msgstr "独立Gamepad视图"
+msgstr "&单独 GamePad 窗口"
 
 #: MainWindow.cpp
 msgid "&General settings"
-msgstr "通用设置"
+msgstr "&通用设置"
 
 #: MainWindow.cpp
 msgid "&Input settings"
-msgstr "输入设置"
+msgstr "&输入设置"
 
 #: MainWindow.cpp
 msgid "&Active account"
-msgstr "当前账户"
+msgstr "&当前账户"
 
 #: MainWindow.cpp
 msgid "&Console region"
-msgstr "主机地区"
+msgstr "&主机地区"
 
 #: MainWindow.cpp
 msgid "&Console language"
-msgstr "主机语言"
+msgstr "&主机语言"
 
 #: MainWindow.cpp
 msgid "&Memory searcher"
-msgstr "内存搜索"
-
-#: MainWindow.cpp
-msgid "&Tools"
-msgstr "工具"
-
-#: MainWindow.cpp
-msgid "&Single-core interpreter"
-msgstr "单核解释器"
-
-#: MainWindow.cpp
-msgid "&Single-core recompiler (fast)"
-msgstr "单核重编译器（快速）"
-
-#: MainWindow.cpp
-msgid "&Dual-core recompiler (fast, unstable!)"
-msgstr "双核重编译器（快速但不稳定）"
+msgstr "&内存搜索器"
 
 #: MainWindow.cpp
-msgid "&Triple-core recompiler (fast, unstable!)"
-msgstr "三核重编译器（快速但不稳定）"
+msgid "&Title Manager"
+msgstr "&档案管理器"
 
 #: MainWindow.cpp
-msgid "&Mode"
-msgstr "模式"
+msgid "&Tools"
+msgstr "&工具"
 
 #: MainWindow.cpp
 msgid "&CPU"
-msgstr "CPU"
+msgstr "&CPU"
 
 #: MainWindow.cpp
 msgid "&Scan NFC tag from file"
-msgstr "载入NFC数据"
+msgstr "&载入 NFC 文件"
 
 #: MainWindow.cpp
 msgid "&NFC"
-msgstr "NFC"
+msgstr "&NFC"
 
 #: MainWindow.cpp
 msgid "&Unsupported API calls"
-msgstr "不支持的API调用"
+msgstr "&不支持的 API 调用"
 
 #: MainWindow.cpp
 msgid "&Coreinit File-Access"
-msgstr "Coreinit 数据访问"
+msgstr "&Coreinit 数据访问"
 
 #: MainWindow.cpp
 msgid "&Coreinit Thread-Synchronization API"
-msgstr "Coreinit 线程同步 API"
+msgstr "&Coreinit 线程同步 API"
 
 #: MainWindow.cpp
 msgid "&Coreinit Memory API"
-msgstr "Coreinit 内存 API"
+msgstr "&Coreinit 内存 API"
+
+#: MainWindow.cpp
+msgid "&Coreinit MP API"
+msgstr "&Coreinit MP API"
+
+#: MainWindow.cpp
+msgid "&NN NFP"
+msgstr "&NN NFP"
 
 #: MainWindow.cpp
 msgid "&GX2 API"
-msgstr "GX2 API"
+msgstr "&GX2 API"
 
 #: MainWindow.cpp
 msgid "&Audio API"
-msgstr "音频 API"
+msgstr "&音频 API"
 
 #: MainWindow.cpp
 msgid "&Input API"
-msgstr "输入 API"
+msgstr "&输入 API"
 
 #: MainWindow.cpp
 msgid "&Socket API"
-msgstr "套接 API"
+msgstr "&套接 API"
 
 #: MainWindow.cpp
 msgid "&Save API"
-msgstr "储存 API"
+msgstr "&储存 API"
 
 #: MainWindow.cpp
 msgid "&H264 API"
-msgstr "H264 API"
+msgstr "&H264 API"
 
 #: MainWindow.cpp
-msgid "&NFP API"
-msgstr "NFP API"
+msgid "&Graphic pack patches"
+msgstr "&图形插件补丁"
 
 #: MainWindow.cpp
 msgid "&Texture cache warnings"
-msgstr "纹理缓存警告"
+msgstr "&纹理缓存警告"
 
 #: MainWindow.cpp
 msgid "&OpenGL debug output"
-msgstr "OpenGL 调试输出"
+msgstr "&OpenGL 调试输出"
 
 #: MainWindow.cpp
 msgid "&Vulkan validation layer (slow)"
-msgstr "Vulkan 验证层 (慢速)"
+msgstr "&Vulkan 验证层 (慢速)"
 
 #: MainWindow.cpp
 msgid "&Log PPC context for API"
-msgstr "为API记录PPC上下文"
+msgstr "&Log PPC context for API"
```

Comment:
Because I am not sure whether this translation is correct. Context in Chinese is usually only used in the article, which means the paragraph before and after, if the translation is correct, of course you can add it.
I will add it, and it won't be too late to change it back if someone gives feedback.